### ISSUE TITLE
The scope of this bundle?

### ISSUE THREAD

**Issue by Nyholm** [php-http/HttplugBundle#10]
I'm curious what your intention of this bundle is. I know it has been a while since this was updated and your focus are on the libraries. At the moment this bundle creates services for the client and factories. It means that it is only good for dependency injection. Is it any other wins atm?

I had an idea of actually be implementation specific here. The bundle config could allow you to configure clients and make sure to create services for a **configured** client. Example:

```yaml
httplug:
  clients:
    acme: 
      adapter: guzzle6
      timeout: 2
      verify_ssl: false

services:
  linkedin:
    class: Happyr\LinkedIn\LinkedIn
    arguments: ["@httplug.acme"]
```

What do you think? Would you accept a pr implementing this?

**Comment by dbu** [comment]
right now the bundle is indeed minimal. i really like your proposal, a PR
is very  welcome.
the other important next step will be to support configuring plugins and eg
handle the symfony debug toolbar

**Comment by Nyholm** [comment]
Awesome. 
I think I can manage some work the next coming days. 

Naturally, we should have integration with the debug bar. :)

**Comment by Nyholm** [comment]
First PR submitted. Next up is debug toolbar and configurable plugins.

**Comment by sagikazarmark** [comment]
Toolbar (data collector) and plugins has their own issue. Can this be closed?

**Comment by Nyholm** [comment]
Sure. 

FYI. I got the toolbar implementation already. Im just going to fix and merge #16

**Comment by sagikazarmark** [comment]
Oh, nice.
